requirements.in

```diff
@@ -3,3 +3,5 @@ datetime
 boto3
 codecov
 requests-futures
+freezegun
+moto
```

Comment:
I'm seeing any reference to moto in your changes, Am I missing something?

Reply:
moto was directly added to `requirement.txt` . I have added same to requirements.in only